.github/workflows/shellcheck.yaml

```
@@ -0,0 +1,15 @@
+on:
+  push
+
+name: 'Shellcheck'
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
+      with:
+        ignore: "./*/gradlew"
```

maybe this should be 
```suggestion
        ignore: "**/gradlew"
```
Looks like it is still failing the build.